test/es5processor_test.ts

```diff
@@ -113,6 +113,12 @@ __export(require('req/mod'));`)
               `goog.module('a');var module = module || {id: 'a.js'};var mod = goog.require('req.mod');
 __export(mod);`);
     });
+
+    it('converts tslib exportStar usage', () => {
+      expectCommonJs('a.js', `tslib_1.__exportStar(require("./decorators"), exports);`)
```

Comment:
what if tslib changes in the future? would be nice to have an end-to-end test also

Reply:
That is a good observation.  There's a chicken and egg problem here where we need to get tslib working end-to-end before we can test it all, so I think I will save it for later.  Ultimately I think we should drop the code that handles the non-tslib codepath because using tslib is always better.  I considered doing that here but I think it will be hard to get everything converted at once.